.github/workflows/ci.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
         with:
           environment-file: environment.yml
           environment-name: openfe
+          cache-env: true
```

Comment:
That's quite neat, might steal that for MDA

Reply:
dude it is amazing, I have it cache env and the downloads so that if there is an env change, at least the download cache will help.

Reply:
it is smart enough to be per-platform as well so it will cache mac, linux, and win builds separately, cache TTL is 1 day so it shound't get so stale it won't catch issues with env solving if upstream changes